Dockerfile

```diff
@@ -43,6 +43,12 @@ RUN \
         python-setuptools   \
         python-pip \
         aptitude \
+        iputils-ping \
+        dnsutils \
+    && \
+    # install ripgrep (rg)
+    curl -L https://github.com/BurntSushi/ripgrep/releases/download/11.0.2/ripgrep_11.0.2_amd64.deb \
+        -o /tmp/ripgrep.deb && dpkg -i /tmp/ripgrep.deb \
```

Comment:
can you add it in the ansible provisionning, here it's just to install what is necessary to then launch ansible :)

Reply:
https://github.com/allan-simon/allan-docker-dev-python/blob/master/provisioning/site.yml